src/main/resources/extendedAliases.txt

```diff
@@ -0,0 +1,49 @@
+#
```

Comment:
This is a lossy change below.  I had added at least another 30 alias, that are not listed here.  Please update the PR with the full list before I merge this PR, if you would.